service/controller/v24/adapter/guest_security_groups.go

```diff
@@ -229,7 +229,7 @@ func getKubernetesAPIRules(cfg Config, hostClusterCIDR string) ([]securityGroupR
 				Description: "Allow traffic from tenant cluster CIDR.",
 				Port:        key.KubernetesAPISecurePort(cfg.CustomObject),
 				Protocol:    tcpProtocol,
-				SourceCIDR:  key.ClusterNetworkCIDR(cfg.CustomObject),
+				SourceCIDR:  key.StatusNetworkCIDR(cfg.CustomObject),
```

Comment:
Is this renaming because this data comes from the Status? 

I find it confusing. I know what the cluster network CIDR is. This makes me think there is also a status network.

Reply:
The naming is probably unfortunate. I aligned it with the AZ getter. I found it weird that the methods are different when they are used in similar places. There was an asymmetry. I think the point back then was to hint to the fact that the information comes from the CR status.